### PR TITLE
fix(style-props): add content property and example to style-props page

### PR DESCRIPTION
**Pull request by aaarichter**
Closes # https://github.com/chakra-ui/chakra-ui/issues/561

## 📝 Description

- add missing `content` property & example for `::before` example with extra note about quotes

## ⛳️ Current behavior (updates)

- content prop is missing and no clear example for its usage

## 🚀 New behavior

- add missing `content` property & example for `::before` example with extra note about quotes

## 💣 Is this a breaking change (Yes/No):

no

### CODE REVIEW

File: pages/docs/features/style-props.mdx [hunk 1]
```diff
@@ -574,6 +574,14 @@ import { Button } from "@chakra-ui/react"
   >
   </Box>
 </Box>
+
+// add ::before pseudo element
+<Box
+  _before={{ content: '"🙂"', display: 'inline-block', me: '5px'} // Note: the content value need an extra set of quotes!
```
Comment (nikolovlazar) [inline pages/docs/features/style-props.mdx:580]: I think you're missing a closing `}` at the end.

**Review by nikolovlazar — CHANGES_REQUESTED** [review]

File: pages/docs/features/style-props.mdx (hunk 1 above)
Comment (nikolovlazar) [reply pages/docs/features/style-props.mdx:580]: Also, which prop is `me`? Were you thinking about `mr` as in `marginRight`?

Comment (aaarichter) [reply pages/docs/features/style-props.mdx:580]: me - marginEnd

I'll add the missing `}`

**Comment by aaarichter** [comment]
@nikolovlazar feedback addressed

File: pages/docs/features/style-props.mdx (hunk 1 above)
Comment (nikolovlazar) [reply pages/docs/features/style-props.mdx]: Also, can you move the Note comment above `_before`, so the code block doesn't scroll horizontally on desktop?

Comment (aaarichter) [reply pages/docs/features/style-props.mdx]: addressed. I force pushed it on the last commit

**Review by jacobhq — APPROVED** [review]
This looks good. Why is the build failing?

**Comment by noobinthisgame** [comment]
@aaarichter when the build is failing in the pipeline, most of the time you can reproduce the fail locally.

**Comment by nikolovlazar** [comment]
@aaarichter we just merged a big PR that restructures the docsite. You have a conflict in the `style-props.mdx` file now, because it got moved to `pages/docs/styled-system/features/style-props.mdx`.

**Comment by aaarichter** [comment]
seems like the rebase did the trick. pipeline is green now. 💚 

> when the build is failing in the pipeline, most of the time you can reproduce the fail locally.

good to know. On that point I was expecting to just open the pipeline tool and see what the issue is but when faced with a login and the need to register, I bailed. I wonder if it is better than github actions, which can be viewed from anyone ?!

**Comment by nikolovlazar** [comment]
@aaarichter just add an empty space after the `'5px'`, and then we merge, okay?

**Comment by aaarichter** [comment]
force pushed @nikolovlazar

**Pull request merged**